app/i18n/en/namespaces/common.js

```diff
@@ -15,11 +15,6 @@ const common = {
     edit: 'Edit',
     delete: 'Delete',
   },
-  link: {
-    topics: 'Topics',
-    newtopic: 'Create new topic',
-    library: 'Library',
-  },
   form: {
```

Comment:
Shouldn't this go under the `topics` module?

Reply:
it could, but then you have duplicate keys of this in global.js

Reply:
For the topics form? If it's a form that relates to topics, it should go under that module. And if there is any form that coincidentally has the same fields, the strings could be duplicated as the functionality is completely independent.

Reply:
My bad. I thought you were referring to the deletions.
That form should indeed go in topics.js